test/components/testDurationFmt.jsx

```diff
@@ -42,4 +41,108 @@ export let testDurationFmt = {
         test.equal(wrapper.text(), '3 hours, 3 minutes, 3 seconds');
         test.done();
     },
+    
+    testDurationFmtLocale: test => {
+        test.expect(1);
+        const duration = {hour: 3, minute: 3, second:3};
+        const wrapper = mount(
+            <LocaleContext.Provider value={{locale: "en-US"}}>
+                <DurationFmt locale="de-DE" length="full" duration={duration}/>
+            </LocaleContext.Provider>
+        );
+
+        test.equal(wrapper.text(), '3 Stunden, 3 Minuten und 3 Sekunden');
+        test.done();
+    },
+
+    testDurationFmtLength: test => {
+        test.expect(1);
+        const duration = {hour: 3, minute: 3, second:3};
+        const wrapper = mount(
+            <LocaleContext.Provider value={{locale: "en-US"}}>
+                <DurationFmt length="short" duration={duration}/>
+            </LocaleContext.Provider>
+        );
+
+        test.equal(wrapper.text(), '3h 3m 3s');
+        test.done();
+    },
+
+    testDurationFmtStyle: test => {
+        test.expect(1);
+        const duration = {hour: 3, minute: 3, second:3};
+        const wrapper = mount(
+            <LocaleContext.Provider value={{locale: "en-US"}}>
+                <DurationFmt style="clock" length="full" duration={duration}/>
+            </LocaleContext.Provider>
+        );
+
+        test.equal(wrapper.text(), '3:03:03');
+        test.done();
+    },
+
+    testDurationFmtUseNative: test => {
+        test.expect(1);
+        const duration = {hour: 3, minute: 3, second:3};
+        const wrapper = mount(
+            <LocaleContext.Provider value={{locale: "en-US"}}>
+                <DurationFmt useNative="true" locale="bn-IN" length="full" duration={duration}/>
+            </LocaleContext.Provider>
+        );
+
+        test.equal(wrapper.text(), '৩ ঘন্টা, ৩ মিনিট, ৩ সেকেন্ড');
+        test.done();
+    },
+
+    testDurationFmtWrapperString: test => {
+        test.expect(1);
+        const duration = {hour: 3, minute: 3, second:3};
+        const wrapper = mount(
+            <LocaleContext.Provider value={{locale: "en-US"}}>
+                <DurationFmt wrapper="span" length="full" duration={duration}/>
```

Comment:
We have a property name `wrapper` and test function has a value name as `wrapper`. Wouldn't it better to give different are not conflicting? And `wrapper ="span"` Does we assume that the developer enter the correct tag name? It look like we don't check if it's a valid tag or not. for example, if I put "spann" iLib returns string includes  "spannn".